act-rules-format.bs

```diff
@@ -315,31 +315,28 @@ The definition of a rule MUST always result in one of the following outcomes:
 </div>
 
 
-Rule Quality Assurance {#quality}
-=================================
+Test Cases {#test-cases}
+========================
 
-Test Cases (Atomic rules only) {#quality-test-cases}
-----------------------------------------------------
-
-Test cases are (snippets of) content that can be used to validate the implementation of an [=atomic rule=]. They consist of two pieces of data, a snippet of each [test aspect](#input-aspects) for a rule, and the [expected result](#output) that should come from that rule. Test cases serve two functions, firstly as example scenarios for readers to understand when a rule passes, when it fails, and when it is inapplicable. But also for developers and users of automated accessibility test tools to validate that a rule is correctly implemented.
+Test cases are (snippets of) content that can be used to validate the implementation of an ACT Rules. Each rule MUST have tefst cases for passed, failed and inapplicable scenarios. A test case consist of two pieces of data, a snippet of each [test aspect](#input-aspects) for a rule, and the [expected result](#output) that should come from that rule. Test cases serve two functions, firstly as example scenarios for readers to understand when a rule passes, when it fails, and when it is inapplicable. But also for developers and users of automated accessibility test tools to validate that a rule is correctly implemented.
```

Comment:
- tefst -> test
- passed, failed and inapplicable -> passed, failed, and inapplicable (oxford comma)
- "that should come from" -> "from applying" (to remove "should")
- "But also for" -> "It also serves" (personal preference)
- "automated accessibility test tools" -> "accessibility testing tools and methodologies" (because this also applies to manual rules)

Reply:
Also in that Test Cases section. The third sentence should read: A test case "consists" of two pieces of data...